modules/assessment/component/createnewassessments.tsx

```diff
@@ -75,19 +75,20 @@ const CreateTemplate = () => {
         options: [''],
         correct_option: 0,
       },
-    ]);
-    updatelistinobj();
+    ];
+    setList(addquestion);
+    updatelistinobj(addquestion);
   };
   const handleRemovequest = () => {
-    setList((list) => {
-      if (list.length > 1) {
-        const updatedList = [...list.slice(0, list.length - 1)];
-        return updatedList;
-      } else {
-        return list;
-      }
-    });
-    updatelistinobj();
```

Comment:
How is your implementation any different from what you changed? coz the previous implementation looks better than yours by react standard

Reply:
Ok sir I'll change it